src/client/service.h

```diff
@@ -104,10 +114,12 @@ class I2PService {
   std::mutex m_HandlersMutex;
 };
 
-/**
- * Simple interface for I2PHandlers.
- * Allows detection of finalization amongst other things.
- */
+/// @class I2PServiceHandler
+/// @brief Simple interface for I2PHandlers. abstract class for handler
+/// Simple interface for I2PHandlers. abstract class for handler
```

Comment:
`Simple interface for I2PHandlers. abstract class for handler`, why repeat the same line?